CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Fix: Do not transform attribute value using attribute type after apply expression plugin JEXL (#1034)
```

Comment:
#1034 is the number of the PR itself. I understand that in this case we don't have any issue, so the PR is cited, right?

Reply:
fixed with issue  number 1034

Reply:
Fixed in c6a1f70